### PR TITLE
Setup Node.js CI with Actions

### PR DESCRIPTION
**Pull request by acostalima**

### CODE REVIEW

**Comment by satazor** [comment]
@acostalima please update the badges to use github and not travis

**Comment by satazor** [comment]
You may also remove `.travis.yml` from the repo as well.

**Comment by acostalima** [comment]
Already removed `.travis.yml` file. Will work on the badges, thanks!

**Comment by acostalima** [comment]
@satazor badged updated!

File: .github/workflows/node-ci.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+name: Node CI
+
+on:
```
Comment (satazor) [inline .github/workflows/node-ci.yml:3]: What's the reasoning to restrict this only on PRs and commits to master? Why not only `on: [push]`? This way we can test CI on those weird bugs that only happen on CI without having to open a PR.

**Comment by satazor** [comment]
Added just a minor comment.

File: .github/workflows/node-ci.yml (hunk 1 above)
Comment (acostalima) [reply .github/workflows/node-ci.yml:3]: You're right. We can get rid of this constraint. Lack of knowledge from my side on setting up CI triggers

Comment (acostalima) [reply .github/workflows/node-ci.yml]: All done! ✅

**Comment by acostalima** [comment]
@satazor as we discussed earlier today, I've removed the cache layer for now. Please have another look. Thanks!

**Pull request merged**